crypto/bn/rsaz_exp_x2.c

```diff
@@ -557,9 +559,9 @@ static void to_words52(BN_ULONG *out, int out_len,
     in_str = (uint8_t *)in;
 
     for (; in_bitsize >= (2 * DIGIT_SIZE); in_bitsize -= (2 * DIGIT_SIZE), out += 2) {
-        out[0] = (*(uint64_t *)in_str) & DIGIT_MASK;
+        out[0] = (*(uint64_t_align1 *)in_str) & DIGIT_MASK;
```

Comment:
Should this be doing a misaligned loop followed by an aligned64 loop?

Reply:
I don't care, and cannot do any testing by myself, I found this annoying error report
just too many times in my e-mail inbox, but it's not under my control when it happens.
This is just to tell the sanitizer that the access may be unaligned, but the
generated code is identical when not sanitizing, so its not worth to speculate.

Reply:
IMO it would be fairly hard to do - please note that in_str does not step by 8 bytes. Also this is not a performance critical part of the code.